index.js

```diff
@@ -65,14 +65,14 @@ hexo.extend.migrator.register('ghost', function(args, callback){
         var postData = {
           title: post.title,
           permalink: post.slug,
-          content: post.markdown,
+          content: JSON.parse(post.mobiledoc).cards[0][1].markdown,
```

Comment:
Ghost decide to move to mobiledoc since version 1.0. The official announcement is [here](https://github.com/TryGhost/Ghost/issues/7429).

However from my opinion Ghost did not follow a standard way to use mobiledoc but stored all data in the `cards[0][1].markdown` part. This behavior seems not been documented anywhere, and what I do is just open and read the exported file.

Reply:
FYI, ran this for my blog migration and it worked with these code updates.